app/views/layouts/application.html.erb

```diff
@@ -13,6 +13,9 @@
     <main class="app-content">
       <%= render partial: "layouts/navbar" %>
       <%= render partial: "layouts/sidebar" %>
+      <div id="flash-message-container">
```

Comment:
Will this double show our flash messages on some pages? LIke the partner#show?

Reply:
I mean, I agree this should be here.

Reply:
Oh good catch. I didnt realize we were acessing `flash` messages in that view (outside the layout). Ill clean that up, (and make sure that the styles look nice.)